nixos-modules/opensearch-dashboards.nix

```diff
@@ -0,0 +1,164 @@
+{ config, lib, options, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.opensearch-dashboards;
+  opt = options.services.opensearch-dashboards;
+
+  ge7 = true;
```

Comment:
Unused?

Reply:
I didn't exactly write these, I made minimum changes to the upstream elasticsearch ones. But yeah, cutting things that aren't used seems great to me.

Reply:
Indeed, but it looks like you also eliminated the usages.